### PR TITLE
create destroy targets and add the custom aplication controller to ma…

### PR DESCRIPTION
**Pull request by leherreras**
As a user I should be able to delete a previously created target so it's no longer available in the map

### CODE REVIEW

File: spec/helpers.rb [hunk 1]
```diff
@@ -11,4 +11,8 @@ def json
   def auth_headers
     @auth_headers ||= user.create_new_auth_token
   end
+
+  def clear_hero
```
Comment (enzofab91) [inline spec/helpers.rb]: Maybe a better name like `restart_unique_generator` or something like that?

**Review by enzofab91 — APPROVED** [review]
Good work!

File: spec/helpers.rb (hunk 1 above)
Comment (leherreras) [reply spec/helpers.rb]: ok, thanks for suggestion

**Review by sebastiancaraballo — APPROVED** [review]
Nice!

**Pull request merged**